sdk/metrics/src/debugEnabledTest/java/io/opentelemetry/sdk/metrics/internal/descriptor/InstrumentDescriptorTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.internal.descriptor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.sdk.metrics.common.InstrumentType;
+import io.opentelemetry.sdk.metrics.common.InstrumentValueType;
+import org.junit.jupiter.api.Test;
+
+/**
+ * {@link InstrumentDescriptor#equals(Object)} must ignore {@link
+ * InstrumentDescriptor#getSourceInfo()}, which only returns a meaningful value when {@code
+ * otel.experimental.sdk.metrics.debug=true}.
+ */
+class InstrumentDescriptorTest {
+
+  @Test
+  void equals() {
```

Comment:
Is it possible to use equalsverifier?

Reply:
I can't figure out how to make it work with an abstract class implemented by autovalue.